.github/workflows/ci-image-diff-sync.yml

```diff
@@ -50,10 +50,10 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-python@v2
       with:
-        python-version: 3.9.9
+        python-version: '3.9.9'
     - uses: actions/setup-node@v2
       with:
-        node-version: 18
+        node-version: '18'
```

Comment:
Even though we have node version specified we are still getting the warning. I'm guessing it's because we are missing quotes? 

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-only-run-job-for-specific-repository

Reply:
I think 16 is the new minimum version, I don't think they mean we should go backwards from 18 to 16.